docs/_source/practical_guides/create_dataset.md

```diff
@@ -52,7 +52,61 @@ fields = [
 The order of the fields in the UI follows the order in which these are added to the `fields` attribute in the Python SDK.
 ```
 
-##### Define `questions`
+#### Define metadata properties
```

Comment:
You're right about questions, but perhaps before guidelines because it's good to make a reference of what the annotation team can find and use within the UI and for that you need to know what metadata will be available for them.

Reply:
I moved it after Questions but left it before Guidelines. I hope this is a good compromise 😄